docs/source/data_analysis/HPC-module-SLEAP.md

```diff
@@ -0,0 +1,610 @@
+```
+
+If the scripts are not in your working directory, you will need to specify their full paths:
+
+```{code-block} console
+$ chmod +x /path/to/train-script.sh
+$ chmod +x /path/to/train_slurm.sh
+```
+:::
+
+Now you can submit the batch script via running the following command
+(in the same directory as the script):
+```{code-block} console
+$ sbatch train_slurm.sh
+Submitted batch job 3445652
+```
+
+You may monitor the progress of the job in various ways:
+
+::::{tab-set}
+
+:::{tab-item} squeue
+
+View the status of the queued/running jobs with [`squeue`](https://slurm.schedmd.com/squeue.html):
+
+```{code-block} console
+$ squeue --me
+JOBID    PARTITION  NAME     USER      ST  TIME   NODES  NODELIST(REASON)
+3445652  gpu        slp_train sirmpila  R   23:11  1      gpu-sr670-20
+```
+:::
+
+:::{tab-item} sacct
+
+View status of running/completed jobs with [`sacct`](https://slurm.schedmd.com/sacct.html):
+
+```{code-block} console
+$ sacct
+JobID           JobName  Partition    Account  AllocCPUS      State ExitCode
+------------ ---------- ---------- ---------- ---------- ---------- --------
+3445652      slp_train        gpu     swc-ac          2  COMPLETED      0:0
+3445652.bat+      batch                swc-ac          2  COMPLETED      0:0
+```
+Run `sacct` with some more helpful arguments.
+For example, you can view jobs from the last 24 hours, displaying the time
+elapsed and the peak memory usage in KB (MaxRSS):
+
+```{code-block} console
+$ sacct \
+  --starttime $(date -d '24 hours ago' +%Y-%m-%dT%H:%M:%S) \
+  --endtime $(date +%Y-%m-%dT%H:%M:%S) \
+  --format=JobID,JobName,Partition,State,Start,Elapsed,MaxRSS
+
+JobID           JobName  Partition      State               Start    Elapsed     MaxRSS
+------------ ---------- ---------- ---------- ------------------- ---------- ----------
+4043595       slp_infer        gpu     FAILED 2023-10-10T18:14:31   00:00:35
+4043595.bat+      batch                FAILED 2023-10-10T18:14:31   00:00:35    271104K
+4043603       slp_infer        gpu     FAILED 2023-10-10T18:27:32   00:01:37
+4043603.bat+      batch                FAILED 2023-10-10T18:27:32   00:01:37    423476K
+4043611       slp_infer        gpu    PENDING             Unknown   00:00:00
+```
+:::
+
+:::{tab-item} view the logs
+
+View the contents of standard output and error
+(the node name and job ID will differ in each case):
+```{code-block} console
+$ cat slurm.gpu-sr670-20.3445652.out
+$ cat slurm.gpu-sr670-20.3445652.err
+```
+:::
+
+::::
+
+```{dropdown} Out-of-memory (OOM) errors
+:color: warning
+:icon: alert-fill
+
+If you encounter out-of-memory errors, there are a few things you can try:
+- Request more CPU memory via the `--mem` argument in the SLURM batch script.
+- Request a specific GPU card type with more GPU memory (e.g. `--gres gpu:a4500:1`). The SWC wiki provides a [list of all GPU card types and their specifications](https://wiki.ucl.ac.uk/display/SSC/CPU+and+GPU+Platform+architecture).
```

Comment:
Is it worth clarifying that the memory reflected by `--mem` and GPU card are not the same?

Reply:
I reworderd this part, hope it's clearer now

Reply:
Looks good to me.